sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -192,12 +197,29 @@ void gnrc_rpl_send_DIS(gnrc_rpl_dodag_t *dodag, ipv6_addr_t *destination)
     _gnrc_rpl_send(pkt, NULL, destination, (dodag ? &dodag->dodag_id : NULL));
 }
 
+static bool _gnrc_rpl_check_DIS_validity(gnrc_rpl_dis_t *dis, uint16_t len)
+{
+    uint16_t expected_len = sizeof(*dis) + sizeof(icmpv6_hdr_t);
```

Comment:
Could you test if the build sizes change anything at all if you modify this variable to `static const uint16_t`?

(I think the compiler should be able to notice that there is no need to allocate a stack variable and use the value immediately in the comparison below regardless of `static const` or not, but I don't have any proof)